homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -173,7 +173,6 @@ async def async_create_remote_control(self, during_setup=False):
                 partial(RemoteControl, self._host, self._port, **params)
             )
 
-            self.state = STATE_ON
```

Comment:
why ? it just leaves the entity without any STATE until the update...

you could probably call `_update()` here...